deep-learning-server/examples/darknet.c

```diff
@@ -0,0 +1,498 @@
+#include "darknet.h"
```

Comment:
Is this your code @mofanv or part of the Darknet framework?  (Similar comment applies to all other files in this changeset: what is written by you, what is provided by Darknet, and what comes from Darknet but was modified by you?)

Reply:
Most codes are based darknet, and changes are made by me in order to compile it as wasm binary, and support multiple inputs, etc. Previous YOLO license + our MIT license are added to each file